test/utility/disagg.c

```diff
@@ -44,6 +44,25 @@ testutil_disagg_storage_configuration(TEST_OPTS *opts, const char *home, char *d
 
         testutil_snprintf(disagg_cfg, disagg_cfg_size, TESTUTIL_ENV_CONFIG_DISAGG,
           opts->disagg_mode, opts->disagg_page_log);
+
+        /* Page delta configuration */
+        char deltas_cfg[128] = "";
+
+        if (opts->internal_page_delta != -1) {
```

Comment:
I would rather update the TESTUTIL_ENV_CONFIG_DISAGG to always include internal_page_delta and leaf_page_delta, such that it looks like:
```        
testutil_snprintf(disagg_cfg, disagg_cfg_size, TESTUTIL_ENV_CONFIG_DISAGG,
          opts->disagg_mode, opts->disagg_page_log, opts->leaf_page_delta, opts->internal_page_delta); ```